trace/Sampling.md

```diff
@@ -0,0 +1,32 @@
+# Sampling
+
+This document is about the sampling bit, sampling decision, samplers and how and when does 
+OpenCensus sample traces. A sampled traces is a traces that gets exported via the configured
+exporters.
+
+## Sampling Bit (propagated via TraceOptions)
+
+Sampling bit is always set only at the start of a Span, and is using a `Sampler`
+
+### What kind of samplers does OpenCensus support?
+* `AlwaysSample` - sampler that always makes a "yes" decision every time.
+* `NeverSample` - sampler that always makes a "no" decision every time.
+* `Probability` - sampler that tries to uniformly sample traces with a given probability. When 
+applied to a child `Span` of a **sampled** parent `Span` then it keeps the sampling decision.
```

Comment:
What about a fixed rate sampling? This might bake extrapolation more complicated, but seems useful to maintain a steady flow of traces regardless of traffic flow.

Reply:
suppose this is a matter of required vs optional? I think it is a sensible option

Reply:
I was wondering about something along the line too, is this a list of required samplers or is this an exhaustive list of allowed samplers?

Reply:
I don't want to increase the requirements from the beginning. I agree fixed rate sampling adds a great value. Can you please add an issue to discuss this? Probably we should prototype a reference implementation for that.

Reply:
https://github.com/census-instrumentation/opencensus-specs/issues/9